scripts/finish-release.ts

```diff
@@ -217,10 +217,12 @@ export async function main() {
   }
 
   console.info(`Merging ${upstreamName}/${releaseBranch} into ${mergeBranch}`);
-  git.merge(`${upstreamName}/${releaseBranch}`, { message: 'chore: merge release commit from main to dev' });
-
-  if (!git.isCleanWorkingTree()) {
```

Comment:
should we still keep this conditional?

Reply:
No, it was conditional previously because I was using a custom wrapper around `git`. I brought in `simple-git`, which throws an error if the merge fails due to conflicts. The try/catch here has effectively the same behavior as the `if` statement would have otherwise.